Silicon/TigerlakePkg/Library/IgdOpRegionLib/GopConfigLib.h

```diff
@@ -0,0 +1,1675 @@
+/** @file

```

Comment:
It looks only VBT_HEADER is used in OpRegion.c file. Can we clean-up this file?

Reply:
I removed some of the unused parameters.